tests/test_version.py

```diff
@@ -25,4 +25,8 @@ def test_version_format() -> None:
 
     assert (
         __version__ == installed_version
-    ), f"{__version__} <> {installed_version}, the installed version does not match with the current codebase"
+    ), (
+        f"{__version__} <> {installed_version}, the installed version does not match with the current codebase. "
```

Comment:
```suggestion
        f"The installed version ({installed_version}) does not match the current codebase version ({__version__})."
```

What do you think of something like this, to make it clear which version is installed and which is current